.vitepress/config.mts

```diff
@@ -92,26 +103,27 @@ export default defineConfig({
           ],
         },
         {
-          text: 'FAQs',
+          text: 'Use Cases',
           collapsed: false,
           items: [
-            { text: 'pgvecto.rs vs. pgvector', link: '/faqs/comparison-pgvector' },
-            { text: 'pgvecto.rs vs. specialized vectordb', link: '/faqs/comparison-with-specialized-vectordb' },
-            { text: `Benchmarks`, link: '/faqs/benchmark' },
-          ]
+            { text: 'Hybrid search', link: '/use-case/hybrid-search' },
+            { text: 'Shortening embedding', link: '/use-case/adaptive-retrieval' },
```

Comment:
Do we need to modify the title?

Reply:
I cannot figure out a good title, I think there are few people know adaptive retrieval. Should we use Matryoshka embedding?

Reply:
I changed to adaptive retrieval